Comments/ClientApp/src/components/App/App.js

```diff
@@ -116,6 +117,11 @@ class App extends React.Component<PropsType, StateType> {
 									</LiveMessenger>
 								</Route>
 
+								{/*Lead Info Page*/}
+								<Route exact path="/LeadInformation">
```

Comment:
```suggestion
								<Route exact path="/leadinformation">
```
We should probably change the url to lowercase and maybe with a dash between the words?